rsconnect/main.py

```diff
@@ -1147,7 +1148,7 @@ def deploy_manifest(
         account=account,
         token=token,
         secret=secret,
-        path=file_name,
+        path=file,
```

Comment:
This is for `deploy_manifest()`. In the previous code, the `kwargs` passed to `RSConnectExecutor(**kwargs)` contained both `file` and `file_name`:
https://github.com/rstudio/rsconnect-python/blob/1.23.0/rsconnect/main.py#L1094-L1121

And then when `validate_app_mode()` was called, it would look first for `path`, then `file`, then `file_name`. With that ordering, `file` took precedence over `file_name`.
https://github.com/rstudio/rsconnect-python/blob/1.23.0/rsconnect/api.py#L857-L868

This change makes it more consistent with the previous behavior.